e2e/cypress/integration/account_settings/theme_color/custom_colors_sidebar_styles_spec.js

```diff
@@ -147,3 +147,84 @@ describe('AS14318 Theme Colors - Color Picker', () => {
         });
     });
 });
+
+describe.only('MM-14318 Theme Colors', () => {
```

Comment:
Forgot to remove "only"?

```suggestion
describe.only('AS14318 - Theme Colors - Color Picker', () => {
```

Reply:
done